src/Std/Tactic/BVDecide/Normalize/BitVec.lean

```diff
@@ -262,22 +262,20 @@ attribute [bv_normalize] BitVec.shiftLeft_ofNat_eq
 attribute [bv_normalize] BitVec.ushiftRight_ofNat_eq
 attribute [bv_normalize] BitVec.sshiftRight'_ofNat_eq_sshiftRight
 
+-- Normalize (1#w + ~~~x) to (~~~x + 1#w) to limit the number of symmetries we need for theorems
```

Comment:
I think this is a nice idea, but it would be good if we could be consistent about this normal form, afaict from a quick grep there are two other affected families of rules:
- `add_neg`
- `not_neg`

Reply:
Thanks for spotting that!